as_manager_node:
  ros__parameters:
    generic:
      WCET: 5000000  # ns
      period: 10000000  # ns
      deadline: 10000000  #Â ns
      safetyFeatures: true
    
    debug:
      verboseCallbacks: false
      verboseHal: false

    topics:
      asStateTopic: "/as/status"
      gearUpTopic: "/command/ecu"
      brakeTopic: "/command/brake"
      clutchTopic: "/command/clutch"
      steerTopic: "/command/steer"
      ecuStatusTopic: "/status/ecu"
      resStatusTopic: "/status/res"
      maxonMotorsTopic: "/actuator/status"
      missionSelectedTopic: "/mission/selected"
      stopMessageTopic: "/mission/stop"
      orinOnTopic: "/orin/on"

    thresholds:
      ebsTankPressureThreshold: 5
      brakePressureOneActuatorThreshold: 10
      brakePressureBothActuatorsThreshold: 20
      brakePressureMaxonMotorsThreshold: 6
      unbrakePressureThreshold: 5

    filtering:
      brakePressureFrontAlpha: 0.8
      brakePressureRearAlpha: 0.8
      rpmAlpha: 0.8